### ISSUE TITLE
False positive, removed super interface when class still indirectly extends interface

### ISSUE THREAD

**Issue by lbergelson** [lvc/japi-compliance-checker#56]
We saw a false positive report of source / binary incompatibility 

The class in question directly extended the interfaces Feature and Locatable

We had a hierarchy like this:

```
interface Locatable
interface Feature extends Locatable
class Interval implements Locatable, Feature
```
In a new version we changed Interval so it no longer directly extends Locatable but still extends it through Feature.

We (incorrectly) get the following error:
`High severity: Removed super-interface Locatable.`
